deepinv/loss/sure.py

```diff
@@ -142,14 +142,14 @@ class SurePoissonLoss(Loss):
 
     .. math::
 
-      y = \gamma z \quad \text{with}\quad z\sim \mathcal{P}(\frac{u}{\gamma}), \quad u=A(x).
+      y = \gamma z \quad \text{with}\quad z\sim \mathcal{P}(u*gamma), \quad u=A(x).

```

Comment:
why are you changing the definition of the parameter?

Reply:
In order to be consistent with the Poisson Data fidelity term and the literature in general I think